### PR TITLE
5 create movie info

### LINKED ISSUE #5: Create Movie-info Page
Width: 375px > iPhone X
Use separate CSS file
Pick right colors from Figma
Use px for Text, % for Shapes so we can expand easily later for desktop view.
Use Button Tag for buttons not Div.
Use flexbox  and sections

Modification:
Use Vw not Px for responsive design
line-height to center text vertically instead of padding

### PR DESCRIPTION
**Pull request by MMansour87**
Done creating the page

deleting font file

### CODE REVIEW

**Review by Rola-H — DISMISSED** [review]

File: CSS-Folder/Movie_Info.css [hunk 1]
```diff
@@ -1,54 +1,59 @@
...
     background-color: #210F37;
     margin: 5vw 5vw 5vw 5vw;
     border-radius: 8vw;
     padding: 4vw 0vw 8vw 8vw  ;
     box-sizing: border-box; 
-
-
 }
 h1{
-    font-size: 23px;
```
Comment (MMansour87) [inline CSS-Folder/Movie_Info.css:23]: Try Vw

File: CSS-Folder/Movie_Info.css [hunk 2]
```diff
@@ -1,54 +1,59 @@
...
+    font-size: 6.1vw;
     color: #fff;
     margin-bottom: 1.3vw;
     font-weight: normal;
 
 
 }
 p{
-    color: #FFF;
-    font-size: 20px;
```
Comment (MMansour87) [inline CSS-Folder/Movie_Info.css:32]: Try VW

File: CSS-Folder/Movie_Info.css [hunk 3]
```diff
@@ -1,54 +1,59 @@
...
+    color: rgba(255, 255, 255, 0.7);;
+    font-size: 5.4vw;
     -webkit-text-stroke: .04vw black; 
     padding-right: 5vw;
 }
 h2{
     color: #ECBBDA;
-    font-size: 15px;
```
Comment (MMansour87) [inline CSS-Folder/Movie_Info.css:38]: Try Vw

**Review by MMansour87 — COMMENTED** [review]
Try Vw instead of Px > for more responsive design

**Review by lina-jamal — APPROVED** [review]

**Review by Rola-H — APPROVED** [review]

**Review by MahmoudJD95 — APPROVED** [review]

**Pull request merged**